pkg/validate/dynamic/platformworkloadidentityprofile.go

```diff
@@ -0,0 +1,108 @@
+package dynamic
+
+import (
+	"context"
+	"net/http"
+
+	sdkauthorization "github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/authorization/armauthorization/v3"
+	"github.com/Azure/go-autorest/autorest/azure"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/util/azureclient/azuresdk/armauthorization"
+	"github.com/Azure/ARO-RP/pkg/util/rbac"
+	"github.com/Azure/ARO-RP/pkg/util/stringutils"
+)
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+func (dv *dynamic) ValidatePlatformWorkloadIdentityProfile(ctx context.Context, oc *api.OpenShiftCluster, platformWorkloadIdentityRoles []api.PlatformWorkloadIdentityRole, roleDefinitions armauthorization.RoleDefinitionsClient) error {
+	dv.log.Print("ValidatePlatformWorkloadIdentityProfile")
+
+	platformIdentitiesActionsMap := map[string][]string{}
+
+	requiredOperatorIdentities := []string{}
+	recievedOperatorIdentities := []string{}
```

Comment:
Just a super nit, totally optional:
```
var (
	platformIdentitiesActionsMap map[string][]string{}
	requiredOperatorIdentities []string{}
	recievedOperatorIdentities []string{}
)
```
Since you have three next to each other I think it reads cleaner like this.

Reply:
Now only using platformIdentitiesActionsMap, so don't need it now.